.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "site/themes/docsy"]
   path = site/themes/docsy
   url = https://github.com/google/docsy.git
+[submodule "site/source/_themes"]
+  path = site/source/_themes
+  url = https://github.com/readthedocs/sphinx_rtd_theme.git
```

Comment:
Please check if we can use a fixed commit. This will save us from having the docs broken after the theme is updated.

Reply:
At the moment the commit is fixed [d64dadf](https://github.com/TOsmanov/datumaro/tree/api-documentation-sphinx/site/source), as with the `docsy` theme, and the update is done manually using:

    git submodule update --remote site/source/_themes

If move the theme folder to the repository, contributors will have to clone it regardless of whether they intend to work with the documentation site or not.